packages/frontend/src/view/pages/user/components/UserAssetTable.tsx

```diff
@@ -14,6 +14,7 @@ export interface UserAssetsTableProps {
   assets: UserAssetEntry[]
   starkKey: StarkKey
   type: 'SPOT' | 'PERPETUAL'
+  myOwnAssets?: boolean
```

Comment:
```suggestion
  isMine?: boolean
```
Also, this prop isn't passed by UserPage!